spec/helpers/ajax-helper.ts

```diff
@@ -139,7 +139,7 @@ export class MockXMLHttpRequest {
   onerror: (e: ErrorEvent) => any;
   onprogress: (e: ProgressEvent) => any;
   ontimeout: (e: ProgressEvent) => any;
-  upload: XMLHttpRequestUpload;
+  upload: XMLHttpRequestUpload = Object.create(null);
```

Comment:
Seems it'd good to leave small comments to describe reason for this for further note.